### PR TITLE
8220730: sun.security.provider.SecureRandom default constructor has wrong documentation

### PR DESCRIPTION
**Pull request by bradfordwetmore**
This is to fix some out-of-date information in the javadoc for the SHA1PRNG that has generated a customer bug.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8220730](https://bugs.openjdk.java.net/browse/JDK-8220730): sun.security.provider.SecureRandom default constructor has wrong documentation

### Reviewers
 * [Weijun Wang](https://openjdk.java.net/census#weijun) (@wangweij - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/1461/head:pull/1461`
`$ git checkout pull/1461`

### CODE REVIEW

File: src/java.base/share/classes/sun/security/provider/SecureRandom.java [hunk 1]
```diff
@@ -65,10 +65,14 @@
     private int remCount;
 
     /**
-     * This empty constructor automatically seeds the generator.  We attempt
-     * to provide sufficient seed bytes to completely randomize the internal
-     * state of the generator (20 bytes).  Note, however, that our seed
-     * generation algorithm has not been thoroughly studied or widely deployed.
+     * This empty constructor.
```
Comment (wangweij) [inline src/java.base/share/classes/sun/security/provider/SecureRandom.java:68]: How about "An empty constructor that creates an unseeded SecureRandom object"?

File: src/java.base/share/classes/sun/security/provider/SecureRandom.java [hunk 2]
```diff
@@ -65,10 +65,14 @@
...
+     * <p>
+     * If the initial state has not been provided by the user via a setSeed()
+     * call, on the first call to engineGetBytes this object will call the
+     * SeedGenerator to provide sufficient seed bytes to completely randomize
+     * the internal state of the generator (20 bytes).  Note that the old
+     * threaded seed generation algorithm is provided only as a fallback, and
+     * has not been thoroughly studied or widely deployed.
      *
      * <p>The first time this constructor is called in a given Virtual Machine,
      * it may take several seconds of CPU time to seed the generator, depending
```
Comment (wangweij) [inline src/java.base/share/classes/sun/security/provider/SecureRandom.java:78]: In my understanding this is not true. The constructor will finish in no time, it's the first engineBytes() call that will take several seconds. We may say "this auto seeding process will...".

**Review by wangweij — COMMENTED** [review]
I feel some sentences are still not precise.

File: src/java.base/share/classes/sun/security/provider/SecureRandom.java (hunk 2 above)
Comment (bradfordwetmore) [reply src/java.base/share/classes/sun/security/provider/SecureRandom.java:78]: Are you talking about the line 78?  Yes, you are correct, I didn't update anything beyond the initial comment.  I can update that as well.

File: src/java.base/share/classes/sun/security/provider/SecureRandom.java (hunk 1 above)
Comment (bradfordwetmore) [reply src/java.base/share/classes/sun/security/provider/SecureRandom.java:68]: Sure, that sounds great.

File: src/java.base/share/classes/sun/security/provider/SecureRandom.java (hunk 2 above)
Comment (wangweij) [reply src/java.base/share/classes/sun/security/provider/SecureRandom.java:78]: Yes, the paragraph starting from line 77.

**Review by wangweij — APPROVED** [review]
Looks fine. Thanks.

File: src/java.base/share/classes/sun/security/provider/SecureRandom.java (hunk 2 above)
Comment (bradfordwetmore) [reply src/java.base/share/classes/sun/security/provider/SecureRandom.java]: Got it, thanks for the reviews.

**Pull request closed without merging**